docs/appendix/intro-video-transcript.md

```diff
@@ -118,12 +118,24 @@ source of the failing test.
 > instance has a custom JavaScript iterator method. That's a strange edge case,
 > but it's important for browsers to agree on every detail!
 
+`http://web-platform-tests.live/css/css-transforms/transform-transformed-tr-contains-fixed-position.html`
```

Comment:
https://wpt.fyi/results/css/css-transforms/transform-transformed-tr-contains-fixed-position.html highlights the "reference vs actual" concept a little more here.